File: pgm/ABC_Java/ABC_400_A.java
import java.io.*;
// import java.math.*;
// import java.time.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final int A = Integer.parseInt (reader.readLine());
        reader.close();
        final int one = 1;
        final int fourHundred = 400;
        final int zero = 0;
        System.out.println (fourHundred % A == zero ? fourHundred / A : -one);
        System.exit (0);
    }
}

File: pgm/ABC_Kotlin/ABC_400_A.kt
// import java.time.*

// import kotlin.math.*

const val fourHundred: Int = 400
const val no: Int = -1
const val zero: Int = 0

fun main() {
    val A: Int = readln().toInt()
    if (fourHundred % A == zero) {
        println(fourHundred / A)
    } else {
        println(no)
    }
    kotlin.system.exitProcess(0)
}